project_proposal/tex/mainpage.tex

```diff
@@ -0,0 +1,390 @@
+continuous integration pipeline for development via GitHub Actions and
+continuous delivery via Watchtower.
+
+\section{Methods}
+To keep track of the group work the Scrum framework, with a few buts, is used.
+Scrum consist of multiple artifacts and ceremonies, and in this project product
+roadmap, Scrum meetings, product and sprint backlogs and burndown charts will
+be used. Each sprint will have a duration of two weeks and the issues for the
+sprint backlog will be chosen at the beginning of each sprint, at the Scrum
+meeting. The Scrum meeting will take place each Friday and will be a mixture of
+sprint planning, daily Scrum, and sprint review. To manage Scrum, ZenHub, a
+management solution that can be integrated with GitHub, is used. On ZenHub the
+group will create the roadmap, that acts as a time schedule for the project and
+board, which is where the issues will be handled. A burndown chart for each
+sprint is automatically generated and kept up to date by ZenHub. The board will
+consist of five columns, as seen below.
+
+\begin{table}[H]
+    \begin{tabularx}{\textwidth}{|>{\RaggedRight}p{4cm}|>{\RaggedRight}X|}
+        \hline
+        Name           & Contains                                 \\
+        \hline
+        Product Baklog & Issues                                   \\
+        \hline
+        Sprint Backlog & Issues for the given sprint              \\
+        \hline
+        In Progress    & Issues that is currently being worked on \\
+        \hline
+        Review/QA      & Issues that are pending (or in) review   \\
+        \hline
+        Closed         & Approved issues that have been merged    \\
+        \hline
+    \end{tabularx}
+    \caption{Methods} 
+    \label{table:Methods}
+\end{table} 
+
+\section{Initial Requirement Analysis}
+
+
+\subsection{Summary of requirements}
+The groups proposed solution will adhere to the requirements given by the brewery Refslevbæk Bryghus A/S.
+
+The manufacturing execution system, MES, must be able to control the brewery’s production.
+It must be able to start and stop the production line,
+as well as monitor the production and collect data from the production line.
+The data must be stored for further analysis.
+The MES must be able to keep track of the batches that the new machine is producing,
+as well as collect various data from the machine that is associated with the current batch number.
+After a finished batch production, the MES must be able to produce a batch report.
+The report must contain the following.
+
+\begin{itemize}
+    \item This Batch ID
+    \item Product type
+    \item Amount of products (total, defect and acceptable)
+    \item Amount of time used in the different states
+    \item Logging of temperature over the production time
+    \item Logging of humidity over the production time
+\end{itemize}
+
+The MES/SCADA system must be able to monitor the production and display live relevant data from the machine.
```

Comment:
SCADA = Supervisory control and data acquisition

The MES/SCADA (Supervisory control and data acquisition) system must...